documentation/server_options.md

```diff
@@ -41,10 +41,37 @@ The Fornax Science Console offers a number of software {term}`environments <envi
 They are currently grouped into two base environments.
 You can select a base environment by choosing one of the following container images from the "Environment" dropdown in the screenshot above:
 
--   **Default Astrophysics** (recommended for most use cases) contains many common astronomy software, including those required to run the demo notebooks.
-    This container image is referred to as **fornax-main**.
--   **High-Energy Astrophysics** contains high-energy software, which currently includes HEASoft.
-    Plans exist to add CIAO, XMM-SAS and Fermitools.
-    This container image is referred to as **fornax-hea**.
+(base-environment-default)=
+### **Default Astrophysics**
+- The base Fornax container will be sufficient for many use cases, and provides a pre-installed set of commonly used astronomy tools. This includes the Python environments (managed with _uv_; see {ref}`compute-environments`) required to run the demonstration notebooks.
 
+- This container image is referred to as **fornax-main** - *please note that there is currently also a `Dev Astrophysics' environment option, and we do not recommend its use unless suggested by the support team.*
+
+When a default astrophysics Fornax instance has been launched, the existing software environments can be activated by following the instructions provided in - {ref}`compute-environments-select-python`.
+
+(base-environment-hea)=
+### **High-Energy Astrophysics**
+- The second base environment contains pre-installed versions of software required for the analysis of high-energy (X-ray, Gamma-ray, etc.) observations.
+
+- This container image is referred to as **fornax-hea** - *please note that there is currently also a `Dev High-Energy Astrophysics' environment option, and we do not recommend its use unless suggested by the support team.*
+
+- Fornax-hea currently includes HEASoft, CIAO, and FermiPy environments; XMM-SAS support is under active development, and plans also exist to include an eROSITA-eSASS install.
+
+Once launched, access to Fornax-hea software environments is provided through _micromamba_ (essentially a light-weight replacement for _conda_) environments - you can list available environments with:
```

Comment:
This page is only about the two choices the user has to make in order to start a server: Server Type and Environment (where "environment" == "base environment" == image). This is getting into details of conda environments (!=image) and how to use them, so I think this line and below don't belong here but rather should be covered in https://docs.fornax.sciencecloud.nasa.gov/intro-forsc#view-preinstalled-software or https://docs.fornax.sciencecloud.nasa.gov/compute-environments. At least some of this content is already there.

Reply:
I'm fine with adding whatever info you need to describe HE software I'd just like it to be split into the existing sections. For example, conda info into the existing conda section instead of mixed in here. I commented on #129 and we should talk through that more before doing any reorganization.